lib/util.js

```diff
@@ -122,6 +122,7 @@ function inspect(obj, opts) {
   // default options
   var ctx = {
     seen: [],
+    proxyCache: new WeakMap(),
```

Comment:
Seems a little wasteful to instantiate a new WeakMap that's not used unless showProxy is true.  I'd just set it to undefined here and instantiate it lazily inside formatValue().